tests/queries/0_stateless/03280_low_cardinality_decimal.sql

```diff
@@ -0,0 +1,192 @@
+-- Test for low cardinality decimal
+SET allow_suspicious_low_cardinality_types = 1;
+DROP TABLE IF EXISTS test_low_cardinality_decimal;
+-- Create a table with LowCardinalitLowCardinalityy and Decimal columns
+CREATE TABLE test_low_cardinality_decimal (
+    id UInt32,
+    value LowCardinality(Decimal(10, 2))
+) ENGINE = Memory;
+
+-- Insert some values into the table
+INSERT INTO test_low_cardinality_decimal VALUES
+(1, 123.45),
+(2, 678.90),
+(3, 123.45),
+(4, 678.90),
+(5, 123.45);
+
+-- Select all values to ensure they are inserted correctly
+SELECT * FROM test_low_cardinality_decimal;
```

Comment:
You should add an `ORDER BY` clause to all selects to make sure the order of the returned records are deterministic.